src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -167,7 +167,8 @@ public BuiltInMethods(Core core)
                     {
                         new KeyValuePair<string, ProtoCore.Type>("array", TypeSystem.BuildPrimitiveTypeObject(PrimitiveType.kTypeVar, Constants.kArbitraryRank))
                     },
-                    ID = BuiltInMethods.MethodID.kCount
+                    ID = BuiltInMethods.MethodID.kCount,
+                    MethodAttributes = new MethodAttributes(){Description = "Returns number of items in the specified list."}
```

Comment:
zora will move this description to resources for localization. She will be adding description to all the other methods here.